src/components/Button/ButtonGroup.tsx

```diff
@@ -50,7 +48,6 @@ class ButtonGroup extends UIComponent<Extendable<ButtonGroupProps>, any> {
 
   public static defaultProps = {
     as: 'div',
-    accessibility: buttonGroupBehavior as Accessibility,
```

Comment:
you can either change the default value to defaultBehavior or remove also the `accessibility` prop (line 27). I would leave the decision up to more experienced folks - removing the prop is a breaking change.

Reply:
agree with the concerns expressed - lets introduce `defaultBehavior` as default `accessibility` prop value and merge

Reply:
talked with @jurokapsiar  - as in foreseeable future there is no need to define accessibility behavior for this component, agreed to remove this prop (to better reflect current state of things)